### PR TITLE
Python3 + comment support

### PR DESCRIPTION
**Pull request by willoftw**
A few documentation changes and python3 support. The python3 version also handles the comments in mapping files. 

Are you planning to keep maintaining this repo? If so I can just archive mine and point to/work on yours as and when required. Or if you are not planning too, I can take the reins and do it the other way around. Users are getting confused I think :).

### CODE REVIEW

**Comment by MiCkSoftware** [comment]
Hi,

BTW feel free to integrate my fork. I think my pull request was never answered....
This tool is so useful and there are so many forks...

Cheers.
Mick.

**Comment by backslash-f** [comment]
This looks good and works fine for me (Python 3).

I'm actually going ahead merging this one and I'll remove the old `shockemu.py`, because there's no need to support two versions of Python.

Question @willoftw, do you have to disable SIP in your macOS in order for this to work?

File: Readme.md [hunk 1]
```diff
@@ -8,8 +8,6 @@
 
 # Setup
 ```zsh
-git clone git@github.com:backslash-f/ShockEmu.git
-cd ShockEmu
```
Comment (backslash-f) [inline Readme.md:12]: Any particular reason for removing these two lines?

**Comment by backslash-f** [comment]
@willoftw Regarding `Are you planning to keep maintaining this repo?`
Yes, I was planning to get an Xbox and get rid of this, but I've change my mind... I'm a PS guy after all. 😁 
Anyway, I'm also opening `Discussions` and `Issues` on this repo.

File: Readme.md (hunk 1 above)
Comment (willoftw) [reply Readme.md:12]: Just a personal preference not to have any direct url links or references to file structures in documentation, they can (and eventually will) change. Hopefully if people have found the repo on github, they know how to clone it :P

**Comment by willoftw** [comment]
@backslash-f okay no worries, ill depreciate mine and point to yours if thats okay?

File: Readme.md (hunk 1 above)
Comment (backslash-f) [reply Readme.md:12]: Makes sense.

**Comment by backslash-f** [comment]
> @backslash-f okay no worries, ill depreciate mine and point to yours if thats okay?

Yes, that sounds ok. 👍🏻

**Pull request merged**